TheWarWithin/Priorities/ShamanRestoration.simc

```diff
@@ -18,10 +20,29 @@ actions+=/bag_of_tricks
 actions+=/surging_totem,if=talent.acid_rain
 actions+=/healing_rain,if=!moving&talent.acid_rain
 actions+=/flame_shock,cycle_targets=1,if=active_enemies<3&refreshable
-# actions+=/primordial_wave
+actions+=/primordial_wave
```

Comment:
This should be healing only with the setting turned on, no?  There's no DPS PWave for Resto any more, I thought.

Reply:
ah, yes you're right.